src/client/Microsoft.Identity.Client/Internal/Broker/BrokerResponseConst.cs

```diff
@@ -17,9 +17,15 @@ internal class BrokerResponseConst
         public const string Bearer = "Bearer";
         public const string CorrelationId = "correlation_id";
         public const string Scope = "scope";
+        public const string AndroidScopes = "scopes";
         public const string ExpiresOn = "expires_on";
         public const string ClientInfo = "client_info";
 
         public const string iOSBrokerNonce = "broker_nonce"; //in response from iOS Broker
+
+        public const string Account = "mAccount";
```

Comment:
>m [](start = 39, length = 1)

qns: should this have an "m"?

Reply:
Yes, this is based on the android broker spec